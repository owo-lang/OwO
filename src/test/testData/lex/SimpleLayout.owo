module OwO.SimpleLayout where

topModule = 233

module SubmoduleA where
  subModuleA = 233

module SubmoduleB where
  subModuleB = 233

module SubmoduleC where
  subModuleC = 233

  module SubmoduleD where
    subModuleD = 233

topModuleAgain = 666
