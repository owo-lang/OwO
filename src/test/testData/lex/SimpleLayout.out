ModuleToken (0 1 1) (6 1 7)
IdentifierToken "OwO" (7 1 8) (10 1 11)
DotToken (10 1 11) (11 1 12)
IdentifierToken "SimpleLayout" (11 1 12) (23 1 24)
WhereToken (24 1 25) (29 1 30)
BraceLToken (31 3 1) (31 3 1)
IdentifierToken "topModule" (31 3 1) (40 3 10)
EqualToken (41 3 11) (42 3 12)
IntegerToken 233 (43 3 13) (46 3 16)
SemicolonToken (48 5 1) (48 5 1)
ModuleToken (48 5 1) (54 5 7)
IdentifierToken "SubmoduleA" (55 5 8) (65 5 18)
WhereToken (66 5 19) (71 5 24)
BraceLToken (74 6 3) (74 6 3)
IdentifierToken "subModuleA" (74 6 3) (84 6 13)
EqualToken (85 6 14) (86 6 15)
IntegerToken 233 (87 6 16) (90 6 19)
BraceRToken (92 8 1) (92 8 1)
SemicolonToken (92 8 1) (92 8 1)
ModuleToken (92 8 1) (98 8 7)
IdentifierToken "SubmoduleB" (99 8 8) (109 8 18)
WhereToken (110 8 19) (115 8 24)
BraceLToken (118 9 3) (118 9 3)
IdentifierToken "subModuleB" (118 9 3) (128 9 13)
EqualToken (129 9 14) (130 9 15)
IntegerToken 233 (131 9 16) (134 9 19)
BraceRToken (136 11 1) (136 11 1)
SemicolonToken (136 11 1) (136 11 1)
ModuleToken (136 11 1) (142 11 7)
IdentifierToken "SubmoduleC" (143 11 8) (153 11 18)
WhereToken (154 11 19) (159 11 24)
BraceLToken (162 12 3) (162 12 3)
IdentifierToken "subModuleC" (162 12 3) (172 12 13)
EqualToken (173 12 14) (174 12 15)
IntegerToken 233 (175 12 16) (178 12 19)
SemicolonToken (182 14 3) (182 14 3)
ModuleToken (182 14 3) (188 14 9)
IdentifierToken "SubmoduleD" (189 14 10) (199 14 20)
WhereToken (200 14 21) (205 14 26)
BraceLToken (210 15 5) (210 15 5)
IdentifierToken "subModuleD" (210 15 5) (220 15 15)
EqualToken (221 15 16) (222 15 17)
IntegerToken 233 (223 15 18) (226 15 21)
BraceRToken (228 17 1) (228 17 1)
BraceRToken (228 17 1) (228 17 1)
SemicolonToken (228 17 1) (228 17 1)
IdentifierToken "topModuleAgain" (228 17 1) (242 17 15)
EqualToken (243 17 16) (244 17 17)
IntegerToken 666 (245 17 18) (248 17 21)
SemicolonToken (249 18 1) (249 18 1)
BraceRToken (249 18 1) (249 18 1)
